src/components/EditNote/edit-note.css

```diff
@@ -0,0 +1,94 @@
+:root {
+  --color-black: black;
+  --note-bg-color: #27292f;
+  --color-white: #ffffff;
+  --color-gray: #808080;
+}
+.edit-note {
+  border: 1px solid var(--color-black);
+  width: 37.5rem;
+  padding: 0.5rem;
+  background-color: var(--note-bg-color);
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  position: absolute;
+  bottom: 8rem;
+  right: 15rem;
+  border-radius: 0.5rem;
+  box-shadow: rgba(28, 28, 33, 0.2) 0px 7px 29px 0px;
+}
+
+.edit-note .note-input {
+  display: flex;
+  padding: 0.5rem;
+  background-color: transparent;
+  border: none;
+  outline: none;
+  font-size: 1.3rem;
+  color: var(--color-white);
+  font-weight: 600;
+}
+
+.edit-note .note-input::placeholder {
+  color: var(--color-gray);
+}
+
+.edit-note .note-head {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.edit-note .note-text-area {
+  width: 100%;
+  height: 6.25rem;
+  background-color: transparent;
+  outline: none;
+  color: var(--color-white);
+  padding: 0.5rem;
+  font-size: 1.3rem;
+  border: none;
+}
+
+.edit-note .note-body {
+  display: flex;
+}
+
+.edit-note .note-footer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0.5rem;
+}
+
+.edit-note .note-icons {
+  display: flex;
+  gap: 2rem;
+}
+
+.edit-note .note-actions {
+  display: flex;
+  gap: 1rem;
+}
+
+.edit-note .note-btn {
+  background: var(--color-white);
+  color: var(--color-black);
+  outline: none;
+  border: none;
+  padding: 0.2rem 0.6rem;
+  font-weight: 600;
+  display: flex;
+  align-items: center;
+  border-radius: 0.5em;
+}
+
+select {
+  background-color: transparent;
+  color: var(--color-white);
+}
+
+option {
+  background-color: var(--color-black);
+}
```

Comment:
CSS properties are repeated, it is better to keep the common CSS in single file.